### PR TITLE
2936 Create Resource button missing for two resource types

### PR DESCRIPTION
**Pull request by engineSound**
### Pull Request Checklist: 
- [ ] Positive Test Case Written by Dev

- [ ] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [ ] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
1. Run through all types of resource type, click create button, verified a resource is created.

### CODE REVIEW

File: hs_core/templates/pages/create-resource.html [hunk 1]
```diff
@@ -405,14 +405,13 @@ <h3>Drop files here or click to upload</h3>
...
-                    <div class="col-xs-12">
-                        <span class="btn btn-primary btn-create-resource pull-right">Create Resource</span>
-                        <span class="btn btn-primary btn-cancel-create-resource pull-left">Cancel</span>
-                    </div>
-                </div>
             </div>
+            
+            <div class="col-xs-12">
+                    <span class="btn btn-primary btn-create-resource pull-right">Create Resource</span>
+                    <span class="btn btn-primary btn-cancel-create-resource pull-left">Cancel</span>
```
[the remark this answers is not in the document]
Comment (engineSound) [reply hs_core/templates/pages/create-resource.html]: What's the difference between btn-default and btn-primary?

**Comment by sblack-usu** [comment]
Is this ready to go?  I'd like to get this deployed on beta ASAP.

File: hs_core/templates/pages/create-resource.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (aphelionz) [reply hs_core/templates/pages/create-resource.html]: The color

[the remark this answers is not in the document]
Comment (engineSound) [reply hs_core/templates/pages/create-resource.html]: Right. The color becomes grey. Thanks.

**Comment by engineSound** [comment]
@Maurier @aphelionz I have made the change.

**Comment by Maurier** [comment]
Looks good.

**Comment by engineSound** [comment]
@aphelionz change made. Please approve the review.

**Pull request merged**